app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -24,8 +24,7 @@ def authentication_status_ok?(type = nil)
   end
 
   def use_graph_refresh?
-    Settings.ems_refresh.rhevm.try(:[], :inventory_object_refresh) &&
-      use_ovirt_sdk? && supported_api_versions.include?('4')
+    true
```

Comment:
@borod108 can we remove this method and update all callers to just do what they would have done for the graph refresh case?

Reply:
I could, I was thinking to do that together with removing tons of other code - you can see I only removed the specs to keep it safe...
Can we leave this to the big refactoring PR?

Reply:
Okay